codec-http2/src/main/java/io/netty5/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -193,7 +193,7 @@ private boolean isChannelWritable() {
     }
 
     private boolean isChannelWritable0() {
-        return ctx.channel().isWritable();
+        return ctx.channel().writableBytes() > 0;
```

Comment:
This can now be reverted.

```suggestion
        return ctx.channel().isWritable();
```